code/datums/diseases/Cancer

```diff
@@ -0,0 +1,45 @@
+/datum/disease/Cancer
+	form = "Condition"
+	name = "Cancer"
+	max_stages = 5
+	cure_text = "Surgery"
+	agent = "Cancer Cells"
+	viable_mobtypes = list(/mob/living/carbon/human)
+	permeability_mod = 1
+	desc = "If left untreated the subject will become very weak,Massive Organ Failure and Die."
+	severity = "Very Dangerous!"
+	disease_flags = CAN_CARRY|CAN_RESIST
+	spread_flags = NON_CONTAGIOUS
+	visibility_flags = HIDDEN_PANDEMIC
+	required_organs = list(/obj/item/organ/internal/Lungs)
+	bypasses_immunity = TRUE
+	virus_heal_resistant = TRUE
+
+/datum/disease/Cancer/stage_act()
+	..()
+	switch(stage)
+		if(1)
+			if(prob(5))
+				affected_mob.emote("cough")
+		if(2)
+			var/obj/item/organ/internal/Lungs/A = affected_mob.get_int_organ(/obj/item/organ/internal/Lungs)
+			if(A)
+				A.inflamed = 1
+				A.update_icon()
+			if(prob(3))
+				to_chat(affected_mob, "<span class='warning'>You feel Pain in your chest!</span>")
+				affected_mob.adjustOxyLoss(10)
+		if(3)
+			if(prob(1))
+				to_chat(affected_mob, "<span class='warning'>You cant Breathe Normaly!</span>")
+    if(4)
```

Comment:
misses a tab from here to the end. It is not part of the switch statement anymore from here